assets/components/questionsContact/questionsContact.jsx

```diff
@@ -33,3 +41,13 @@ export default function QuestionsContact() {
   );
 
 }
+// ----- Default Props ----- //
+
+QuestionsContact.defaultProps = {
+  countryGroupId: 'GBPCountries',
+};
+
+
+// ----- Exports ----- //
+
+export default QuestionsContact;
```

Comment:
👍, as I mentioned above, this is typically where we have an exception in our "exports at the bottom of a file" convention, and put `defaultProps` below the export (as in e.g. [`CtaLink`](https://github.com/guardian/support-frontend/blob/master/assets/components/ctaLink/ctaLink.jsx#L54)).

However, that's probably not a good exception and maybe we should adopt what you've done here as the new convention for consistency. @svillafe what do you think?